mattermost-helm/values.yaml

```diff
@@ -171,7 +167,7 @@ minio:
   secretKey: mattermostadmin
   nodeSelector: {}
   persistence:
-    enabled: true
+    enabled: false
```

Comment:
We should have reasonable defaults in the `values.yaml` not defaults setup for our build server. We should modify this just for the test.

Reply:
Ok, but in my opinion, all persistence should set to false. 2/5. I will revert that and set in the test for now

Reply:
Ok. But this PR is about setting up the build, not changing defaults, so you can see why I thought it was related.